npm-shrinkwrap.json

```diff
@@ -86,7 +86,6 @@
       "version": "7.0.0",
       "resolved": "https://registry.npmjs.org/@babel/helper-annotate-as-pure/-/helper-annotate-as-pure-7.0.0.tgz",
       "integrity": "sha512-3UYcJUj9kvSLbLbUIfQTqzcy5VX7GRZ/CCDrnOaZorFFM01aXp1+GJwuFGV4NDDoAS+mOUyHcO6UD/RfqOks3Q==",
-      "dev": true,
```

Comment:
The `dev: true` removals seem strange, did you use npm 6.7.0 ?

Reply:
Yes I did, I'm now running the installation step again (also `@types/react-dom` was missing)

Reply:
Same results